mods/ts/rules/nod-vehicles.yaml

```diff
@@ -36,7 +36,7 @@ BIKE:
 		Cost: 600
 	Tooltip:
 		Name: Attack Cycle
-		Description: Fast scout vehicle, armed with \nrockets.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
+		Description: Fast scout vehicle, armed with\nrockets.\n  Strong vs Vehicles, Aircraft\n  Weak vs Infantry
```

Comment:
This is wrong. The Bike's missiles are currently anti-ground only (same as in the original), so it should be just 'Strong vs Vehicles'.